src/github.ts

```diff
@@ -466,6 +497,77 @@ export class GitHub {
     return refResponse.data.object.sha;
   }
 
+  /**
+   * Find the most "last" merged PR given a headBranch. "last" here means
+   * the most recently created. Includes all associated files.
+   *
+   * @param {string} headBranch - e.g. "release-please/branches/main"
+   * @returns {MergedGitHubPRWithFiles} - if found, otherwise undefined.
+   */
+  async lastMergedPRByHeadBranch(
```

Comment:
sorry, yet another way of querying for the last merged release PR but I wanted something specifically tailored to the manifest/aggregate use case. It's not apparent yet in this PR but the follow on PR for `Manifest.githubRelease` uses the associated PR files to determine which packages actually released.

In the case that no last merged PR is found, the manifest releaser will fall back to looking at the tip of the defaultBranch for package version info. This should handle bootstrapping cases gracefully - both first time release-please users as well as cutting an LTS branch and configuring another release-please on it